src/NUnitFramework/framework/Internal/TestFilter.cs

```diff
@@ -94,7 +94,7 @@ public virtual bool IsExplicitMatch(ITest test)
         /// </summary>
         /// <param name="test">The test to which the filter is applied</param>
         /// <returns>True if the filter matches the an ancestor of the test</returns>
-        private bool MatchParent(ITest test)
+        public bool MatchParent(ITest test)
```

Comment:
None, that I am aware of.  But that does not really make a difference.  It could, in the future.

And we do need to access it here.